src/main/java/com/microfocus/application/automation/tools/octane/CIJenkinsServicesImpl.java

```diff
@@ -667,7 +672,7 @@ private PipelineNode createPipelineNodeFromJobName(String name) {
 
     private void addParametersAndDefaultBranchFromConfig(Item item, PipelineNode result) {
         String defaultBranchesConfig = config != null ? config.getDefaultBranches() : null;
-        if(defaultBranchesConfig !=null && !defaultBranchesConfig.isEmpty()) {
+        if(StringUtils.isNotEmpty(defaultBranchesConfig)) {
```

Comment:
please revert it because this is not passing the code-climate validation run over our code

Reply:
@nissimshitrit I have reverted that, thanks)